### ISSUE TITLE
Unable to add local modifications to firejail profiles without modifying the derivation

### ISSUE THREAD

**Issue by kisik21** [NixOS/nixpkgs#39757]
## Issue description
Cannot modify firejail configurations for, for example, firefox, because .local file includes are pointed to /nix/store.

### Steps to reproduce
`nix-env -iA nixos.firejail && grep local /nix/store/*-firejail-*/etc/firejail/firefox.profile`

### Workaround
Replace includes to .local file to /etc/firejail/{name}.local. Could be done with sed maybe?

## Technical details
I don't think it's relevant here, but still...
 - system: `"x86_64-linux"`
 - host os: `Linux 4.14.35, NixOS, 18.03.132083.06c576b0525 (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0`
 - channels(root): `"nixos-18.03.132083.06c576b0525"`
 - channels(kisik21): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by yurkobb** [comment]
Is this still the case? Maybe it depends on profile? If I inspect `/nix/store/r766faki9abhsgx18ylrmqkh5ljxk512-firejail-0.9.62/etc/firejail/telegram-desktop.profile` I see no hard-coded path to `/nix/store`, just relative paths plus reference to `/etc/firejail/telegram.local`. The version of firejail on my system (20.03) is 0.9.62.

I am able to declaratively override things by doing something like:

```nix
  # Further restrict Telegram in Firejail
  environment.etc."firejail/telegram.local".text = ''
    whitelist ''${HOME}/Downloads
    whitelist ''${HOME}/Desktop
    whitelist ''${HOME}/.TelegramDesktop
    whitelist ''${HOME}/.local/share/TelegramDesktop
  '';

```

**Comment by yurkobb** [comment]
Overriding using `/etc` seems to be fixed since [this commit](https://github.com/NixOS/nixpkgs/commit/0778f0aee6d703b68a2b1ea0ffe1c4c81b6f4a1c), however this means the '*.local' profile paths are now hardcoded to /etc, and the ones in `~/.config/firejail/*.local` are thus ignored.

This seems to deviate from the behavior described [in firejail wiki](https://github.com/netblue30/firejail/wiki/Creating-Profiles):

> Therefore the easiest way to add one or more commands to a profile, is to create a .local file in ~/.config/firejail/ and write the new commands to it.

By hardcoding to `/etc` we are thus breaking the upstream behavior.

Maybe the correct thing to do would be to leave the `.local` includes without a prefix path at all? Just the file name?

**Comment by ryneeverett** [comment]
Resolved by #83515.

**Comment by ryneeverett** [comment]
@yurkobb Looks like the patch was merged after 20.03 was branched.
